base/multidimensional.jl

```diff
@@ -691,9 +691,9 @@ function _cumsum!(out, v, axis, ::TypeArithmetic)
 end
 
 """
-    cumsum(A, dim)
+    cumsum(A, axis::Integer)
```

Comment:
I'd rather keep `dim` or use `region`, which are the two terms that appear to be common in similar functions.

Reply:
This is what the method uses. They used to be out of sync. However, since we have `reducedim`, it is probably better to use `dim` here as well. I'll update.